skflow/io/data_feeder.py

```diff
@@ -183,14 +195,17 @@ def get_feed_dict_fn(self, input_placeholder, output_placeholder):
             from X and y.
         """
         def _feed_dict_fn():
-            inp = np.zeros(self.input_shape, dtype=self.input_dtype)
+            # take random indices
+            batch_indices = self.indices[self.offset: self.offset+self.batch_size]
```

Comment:
what about if batch_indices is smaller than batch size (e.g. you finished first epoch), take rest of the data from the beginning (e.g. circular)?